monitoring/views/airflow_dag_run.view.lkml

```diff
@@ -84,13 +96,13 @@ view: +airflow_dag_run {
 
   measure: avg_execution_time_sec {
     type: average
-    sql: datetime_diff(${end_raw}, ${start_raw}, second) ;;
+    sql: ${seconds_run_time};;
```

Comment:
I'm just curious where `seconds_run_time` is defined as I don't see it in Looker-Hub or here, is it from [here](https://github.com/mozilla/looker-spoke-default/blob/7763f8719565d4b0040d9b676a7b9c3d0bdccca1/monitoring/views/airflow_dag_run.view.lkml#L24C7-L24C13)?

Reply:
yes it's coming from the group ... thought it would be nicer to group it so that I have all the options and not clutter the explore
does show the correct thing in dev so I think I did it right :)

Reply:
Thanks, LGTM too!